Prox/Prox/Utilities/AppConstants.swift

```diff
@@ -32,18 +30,21 @@ public struct AppConstants {
     public static let BuildChannel: AppBuildChannel = {
         #if MOZ_CHANNEL_CURRENT_LOCATION
             return AppBuildChannel.CurrentLocation
-        #elseif MOZ_CHANNEL_RELEASE
-            return AppBuildChannel.Release
-        #elseif MOZ_CHANNEL_MOCK_LOCATION
-            return AppBuildChannel.MockLocation
         #else
-            return AppBuildChannel.Debug
+            #if !MOZ_CHANNEL_MOCK_LOCATION
```

Comment:
nit: don't waste time changing now but I think it'd be clearer as

if CURRENT_LOC
else if MOCK_LOC
else
  assertionfailure